packages/office-ui-fabric-react/src/components/SearchBox/examples/SearchBox.FullSize.Example.tsx

```diff
@@ -3,26 +3,25 @@ import { SearchBox } from 'office-ui-fabric-react/lib/SearchBox';
 import { Stack } from 'office-ui-fabric-react/lib/Stack';
 
 // tslint:disable:jsx-no-lambda
-export class SearchBoxFullSizeExample extends React.Component<any, any> {
-  public render(): JSX.Element {
-    return (
-      <Stack tokens={{ childrenGap: 20 }}>
-        <SearchBox
-          placeholder="Search"
-          onSearch={newValue => console.log('value is ' + newValue)}
-          onFocus={() => console.log('onFocus called')}
-          onBlur={() => console.log('onBlur called')}
-          onChange={() => console.log('onChange called')}
-        />
-        <SearchBox
-          placeholder="Search with no animation"
-          onSearch={newValue => console.log('value is ' + newValue)}
-          onFocus={() => console.log('onFocus called')}
-          onBlur={() => console.log('onBlur called')}
-          onChange={() => console.log('onChange called')}
-          disableAnimation
-        />
-      </Stack>
-    );
-  }
-}
+
+export const SearchBoxFullSizeExample = () => {
+  return (
+    <Stack tokens={{ childrenGap: 20 }}>
+      <SearchBox
+        placeholder="Search"
+        onSearch={newValue => console.log('value is ' + newValue)}
```

Comment:
For the same reason, all these callbacks should also go in constants near the top or bottom of the file, outside the component (and you can remove the tslint:disable).
```ts
const onSearch = (newValue: any) => console.log(`searched for "${newValue}"`);
const onFocus = (ev: React.FocusEvent<HTMLInputElement>) => console.log('onFocus called');
const onBlur = (ev: React.FocusEvent<HTMLInputElement>) => console.log('onBlur called');
const onChange = (ev: React.ChangeEvent<HTMLInputElement>) => console.log(`onChange called. Value: "${ev.target.value}"`);
```